ext/opentelemetry-ext-grpc/setup.cfg

```diff
@@ -50,3 +50,6 @@ test =
 
 [options.packages.find]
 where = src
+
+opentelemetry_instrumentor =
+grpc = opentelemetry.ext.grpc:GrpcInstrumentor
```

Comment:
Not sure how to do this properly, but I think this needs to set up GrpcInstrumentorServer and GrpcInstrumentorClient, there is no such thing as GrpcInstrumentor

Reply:
Yes that's correct. One thing you could do here is to implement a single GrpcInstrumentor that calls instrument for both GrpcInstrumentorServer and GrpcInstrumentorServer. Alternatively you could define two values here, one for the client and one for the server